test/query.spec.js

```diff
@@ -1,429 +1,291 @@
+              'select distinct ?s ?sLabel ?sType ?sTypeLabel ?p ?pLabel ?o ?oLabel ?oType ?oTypeLabel where {\n' +
+              '  VALUES ?basicProvProperties { prov:wasGeneratedBy prov:wasDerivedFrom prov:wasAttributedTo prov:startedAtTime prov:used prov:wasInformedBy prov:endedAtTime prov:wasAssociatedWith prov:actedOnBehalfOf prov:alternateOf prov:specializationOf prov:generatedAtTime prov:hadPrimarySource prov:value prov:wasQuotedFrom prov:wasRevisionOf prov:invalidatedAtTime prov:wasInvalidatedBy prov:hadMember prov:wasStartedBy prov:wasEndedBy prov:invalidated prov:influenced prov:atLocation prov:generated prov:wasInfluencedBy prov:qualifiedInfluence prov:qualifiedGeneration prov:qualifiedDerivation prov:qualifiedPrimarySource prov:qualifiedQuotation prov:qualifiedRevision prov:qualifiedAttribution prov:qualifiedInvalidation prov:qualifiedStart prov:qualifiedUsage prov:qualifiedCommunication prov:qualifiedAssociation prov:qualifiedEnd prov:qualifieddropgation prov:influencer prov:entity prov:hadUsage prov:hadGeneration prov:activity prov:agent prov:hadPlan prov:hadActivity prov:atTime prov:hadRole }\n' +
+              '         <http://www.example.org#chart2> (prov:wasGeneratedBy | prov:wasDerivedFrom | prov:wasAttributedTo | prov:startedAtTime | prov:used | prov:wasInformedBy | prov:endedAtTime | prov:wasAssociatedWith | prov:actedOnBehalfOf | prov:alternateOf | prov:specializationOf | prov:generatedAtTime | prov:hadPrimarySource | prov:value | prov:wasQuotedFrom | prov:wasRevisionOf | prov:invalidatedAtTime | prov:wasInvalidatedBy | prov:hadMember | prov:wasStartedBy | prov:wasEndedBy | prov:invalidated | prov:influenced | prov:atLocation | prov:generated | prov:wasInfluencedBy | prov:qualifiedInfluence | prov:qualifiedGeneration | prov:qualifiedDerivation | prov:qualifiedPrimarySource | prov:qualifiedQuotation | prov:qualifiedRevision | prov:qualifiedAttribution | prov:qualifiedInvalidation | prov:qualifiedStart | prov:qualifiedUsage | prov:qualifiedCommunication | prov:qualifiedAssociation | prov:qualifiedEnd | prov:qualifiedDelegation | prov:influencer | prov:entity | prov:hadUsage | prov:hadGeneration | prov:activity | prov:agent | prov:hadPlan | prov:hadActivity | prov:atTime | prov:hadRole)+ ?o .\n' +
+              '  ?s ?basicProvProperties ?o .\n' +
+              '  ?s ?p ?o .\n' +
+              '  OPTIONAL{ ?s rdfs:label ?sLabel }\n' +
+              '  OPTIONAL { ?p rdfs:label ?pLabel } \n' +
+              '  OPTIONAL{ ?o rdfs:label ?oLabel }\n' +
+              '  OPTIONAL { ?s rdf:type ?sType .\n' +
+              '            OPTIONAL { ?sType rdfs:label ?sTypeLabel . }\n' +
+              '           }\n' +
+              '  OPTIONAL { ?o rdf:type ?oType .\n' +
+              '            OPTIONAL { ?oType rdfs:label ?oTypeLabel . }\n' +
+              '           }' +
+              '}',
+        },
+        data => {
+          expect(data.results.bindings).toEqual(expect.anything());
```

Comment:
Would this return true if data.results.bindings equalled {}? Because that's almost certainly not desired